appveyor.yml

```diff
@@ -27,13 +27,21 @@ install:
         throw "There are newer queued builds for this pull request, failing early." }
 
   # set up environment
-  - "set PATH=%MINICONDA%;%MINICONDA%\\Scripts;%PATH%"
+  - CALL "%MINICONDA%\\Scripts\\activate.bat"
   - conda config --set always_yes yes --set show_channel_urls true --set changeps1 no
-  - conda install conda=4.6
+  - conda update conda
+  # this is basically equivalent to what conda init does.  It changes the "conda" to
+  #    be a .bat script that sets appropriate PATH entries before conda hits problems.
```

Comment:
And `conda init` itself does not work on AppVeyor?

Reply:
It does, but I don't know the equivalent command for "source the new script" or otherwise start a new session. There's another way with some come hook command, by I can't remember and I'm having trouble finding it right now.